src/orderbook_fetcher.ts

```diff
@@ -28,7 +29,9 @@ export class OrderbookFetcher {
           await orderbook.update()
         }
         logger.info('Updating orderbooks map...')
-        this.orderbooks = updateOrderbooks(orderbook)
+        const result = updateOrderbooks(orderbook)
+        this.orderbooks = result[0]
+        this.encodedOrders = result[1]
```

Comment:
nit: I would prefer destructuring here, since the return is more of a tuple than an array.
```suggestion
        const [orderbooks, encodedOrdes] = updateOrderbooks(orderbook)
        this.orderbooks = orderbooks
        this.encodedOrders = encodedOrdes
```